mautic/mautic
# Fix IdentifyCompanyHelper  for unique fields of companies

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | "features" for all features, enhancements and bug fixes (until 3.3.0 is released) 
| Bug fix?                               | yes
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | yes
| Related user documentation PR URL      | 
| Related developer documentation PR URL | 
| Issue(s) addressed                     | 




#### Description:

This is regression fix of Add support for unique fields for companies https://github.com/mautic/mautic/pull/9461



#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Create form with email and text field companyaddress1
3. Try submit
4. Return errror 
`mautic.CRITICAL: Uncaught PHP Exception Mautic\LeadBundle\Exception\UniqueFieldNotFoundException:`

After PR should works properly
